app/helpers/form_helper.rb

```diff
@@ -104,27 +104,28 @@ def radio_button_f(f, attr, options = {})
     end
   end
 
-  def select_f(f, attr, array, id, method, select_options = {}, html_options = {})
+  def select_include_blank(array, include_blank, html_options, select_options)
     array = array.to_a.dup
-    disable_button = select_options.delete(:disable_button)
-    include_blank = select_options.delete(:include_blank)
-    disable_button_enabled = select_options.delete(:disable_button_enabled)
-    user_set = !!select_options.delete(:user_set)
-
     if include_blank
       blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
-      blank_option = OpenStruct.new({id => '', method => blank_value })
-      # if the method is to_s, OpenStruct will respond with its own version.
-      # in this case, I need to undefine its own alias to to_s, and use the attribute
-      # that was defined in the struct.
-      blank_option.instance_eval('undef to_s', __FILE__, __LINE__) if method.to_s == 'to_s' || id.to_s == 'to_s'
+      blank_option = OpenStruct.new({:id => '', :method => blank_value })
+      blank_option.instance_eval('undef to_s', __FILE__, __LINE__)
```

Comment:
couldnt use `{id => '', method => blank_value }`  since ruby complained that it doesnt know what "id" is

Reply:
I tried this, and put costruct_opts in form_select_f and select_f, and the issue is I get the `undefined local variable or method` , but also the error says "Trace of template inclusion: #<ActionView::Template app/views/nic/manageds/_managed.html.erb locals=["f", "id", "managed"]>, #"
so isnt id a local here?